Library/Formula/brew-file.rb

```diff
@@ -0,0 +1,32 @@
+class BrewFile < Formula
+  desc "Brewfile manager for Homebrew."
+  homepage "https://github.com/rcmdnk/homebrew-file/"
+  url "https://github.com/rcmdnk/homebrew-file.git",
+    :tag => "v3.5.9",
+    :revision => "575574b9e6d270dc2f3955df0add6c03ed18b663"
+  head "https://github.com/rcmdnk/homebrew-file.git", :branch => "master"
+  if build.with? "bash"
```

Comment:
We don't really do things like this. What's different about the `bash` branch? Or rather, what gets installed from there that wouldn't be installed otherwise?

I think what you're looking to do here is create a `resource` block, but it's not clear.